modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,199 @@
+:_module-type: PROCEDURE
+
+[id="configuring-your-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring VS Code Server
+
+Incorporating standard database clients into your VS Code Server environment can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your Code Server workbench through built-in VS Code extensions or by creating a custom notebook image using a Dockerfile.
```

Comment:
We should incorporate a link to the project we actually use for this here - https://github.com/coder/code-server/releases

Reply:
Updated. Added a link in the description of the image in notebook-images-for-data-scientists as well.